src/index.ts

```diff
@@ -415,11 +511,12 @@ const extension: JupyterFrontEndPlugin<void> = {
 
     const createAuthLoginComponentCommand = 'cloud-dataproc-settings:configure';
     commands.addCommand(createAuthLoginComponentCommand, {
-      label: 'Cloud Dataproc Settings',
+      label: 'Google BigQuery Settings',
```

Comment:
do we want to rename this setting already?

Reply:
Resolved